CMakeLists.txt

```diff
@@ -17,10 +17,34 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   set(add_link_deps Threads::Threads)
 endif()
 
+##############
+# Installation paths
+set(INCLUDE_INSTALL_DIR include/ CACHE PATH "Installation directory for C++ headers")
+set(LIB_INSTALL_DIR lib${LIB_SUFFIX}/ CACHE PATH "Installation directory for libraries")
+
```

Comment:
From line 20 through 23 should be moved after line 31 since nobody refers to the variables.